docs/HowToGuides/GettingStarted.md

```diff
@@ -350,7 +350,7 @@ while retaining the option of building with Ninja on the command line.
 
 Assuming that you have already [built the toolchain via Ninja](#the-actual-build),
 several more steps are necessary to set up this environment:
-* Generate Xcode projects with `utils/build-script --xcode --swift-darwin-supported-archs "$(uname -m)"`.
+* Generate Xcode projects with `utils/build-script --release --swift-darwin-supported-archs "$(uname -m)" --xcode`.
```

Comment:
Wouldn’t release be harder to debug though due to inlining etc.?

Reply:
`--xcode` doesn’t build anything aside from a subset of LLVM that for reason is required to configure the Xcode projects. The resulting build artifacts are not used for anything else in this workflow, so there’s nothing to run and potentially debug.

Reply:
In that case - great!

Reply:
Does that somehow affect indexing of sources? May it has nothing to do with it, but I did had that impression when I tried. Or maybe was just an impression... Did you experienced something like this?

Reply:
Frankly, I have not tried the release variant *yet*, this just came to my mind spontaneously. Mine’s a RelWithDebInfoAssert (I used to just borrow the `build-script` arguments that I pass for Ninja), and the only indexing issue I’ve faced so far is the one described in the tutorial — when you round-trip Xcode through a distant branch with non-trivial differences in the project structure. I am convinced this is an Xcode caching issue though. Indexing is based on type-checked ASTs, so the state of the optimization switch shouldn’t matter.